src/Feature/Account/client/Integration/Account/reducer.ts

```diff
@@ -100,7 +107,6 @@ export const saveAddressListReducer = (state: SavedAddressListState = { ...addre
   switch (action.type) {
     case reducerActionTypes.ADDRESS_ADD_FAILURE:
     case reducerActionTypes.ADDRESS_ADD_REQUEST:
-    case reducerActionTypes.ADDRESS_ADD_SUCCESS:
```

Comment:
Why ?

Reply:
I added another behavior for this action.